GOVERNANCE.md

```diff
@@ -0,0 +1,206 @@
+
+**Off-boarding Criteria**
+
+An off-boarding vote may be called by any maintainer if any of the following criteria are met:
+- A maintainer has made less than 30 contributions over a span of 1 year.
+  - Contributions will be tracked manually until having an active Devstats Dashboard for the project.
+  - Other relevant data will be collected and evaluated to assess the maintainer's contributions. This includes their involvement in discussions, conversations on Slack, and any other relevant interactions.
+
+The off-boarding process includes the following steps:
+- The off-boarding process is initiated by any currently active maintainer who conducts a review of the maintainers list and proceeds to initialize the off-boarding process if the above criteria are met.
+- The plans of off-boarding process is sent in a private Slack message or email to the candidate.
+- If the candidate for removal states plans to continue participating, another 6 months will be granted to the candidate to make contributions and the new cycle starts. No action is taken and this process terminates.
+- If the candidate fails to meet the criteria during the second attempt to make contributions, the off-boarding process continues.
+- A pull request (PR) proposing movement of the candidate is sent, initiating the public voting phase.
+- The vote passes if a majority of current maintainers vote yes during a voting period lasting five (5) working days.
+- A positive vote will result in movement to an _emeritus_ status within `MAINTAINERS.md` and removal from organization membership.
+
+#### Mapping Project Roles to GitHub Roles
+
+The roles used in this document are custom roles mapped according to the [GitHub roles and responsibilities](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization).
+
+| Project Role   | GitHub Role    |
+| -------------- | -------------- |
+| Contributor    | Triage         |
+| Maintainer     | Maintain       |
+
+### Off-boarding Guidance
+
+If any of the above roles hasn't contributed in any phases (including, but not limited to: code changes, doc updates, issue discussions) in 3 months, the administrator needs to inform the member and remove one's roles and GitHub permissions.
+
+## Maintainer Areas
+
+The k0s projects code base cover many areas and project maintainers are not required to know everything about a project.
+For this reason, maintainers can be specific to one (or more) area of the code base, every area representing a specific aspect.
+
+### k0s Projects
+
+- [k0s](https://github.com/k0sproject/k0s)
+- [k0smotron](https://github.com/k0sproject/k0smotron)
+- [k0sctl](https://github.com/k0sproject/k0sctl)
+- [k0sproject rig](https://github.com/k0sproject/rig)
+- [k0s Community](https://github.com/k0sproject/community)
+- [k0s Website](https://github.com/k0sproject/k0sproject.github.io)
+
+### Projects areas
```

Comment:
I don't get the purpose of this section, the projects listed here seem bit random to me

Reply:
The purpose of this is to highlight the main repos @jnummelin . Not random. We can add more repos if needed. But highlighting main repos is important.